Source/WebCore/rendering/RenderListMarker.cpp

```diff
@@ -344,7 +344,7 @@ void RenderListMarker::computePreferredLogicalWidths()
 
     LayoutUnit logicalWidth;
     if (widthUsesMetricsOfPrimaryFont())
-        logicalWidth = (font.metricsOfPrimaryFont().intAscent() * 2 / 3 + 1) / 2 + 2;
+        logicalWidth = (font.metricsOfPrimaryFont().ascent() * 2 / 3 + 1) / 2 + 2;
```

Comment:
@alanbaradlay - I changed this from your patch. If it is wrong, happy to change.

Reply:
It's not wrong. If you look at the history of this bugzilla, you will see that I've been testing different variants to make this transition as incremental as possible. In order to be able to deal with potential fallout (which is certainly going to happen) in way that we don't have to keep disabling the entire feature and then turn it back, it has to be focused and incremental (see  https://commits.webkit.org/275820@main https://commits.webkit.org/275741@main  https://commits.webkit.org/275624@main and a few more)